src/algebra/order/group/abs.lean

```diff
@@ -62,13 +62,13 @@ begin
 end
 
 lemma eq_or_eq_neg_of_abs_eq {a b : α} (h : |a| = b) : a = b ∨ a = -b :=
-by simpa only [← h, eq_comm, eq_neg_iff_eq_neg] using abs_choice a
+by simpa only [← h, eq_comm, neg_eq_iff_eq_neg] using abs_choice a
```

Comment:
```suggestion
by simpa only [← h, neg_eq_iff_eq_neg] using abs_choice a
```
maybe?

Reply:
This does not work (`abs_choice a` wants the equalities reversed).